### PR TITLE
oryx dotnet 8.0.1 cleanup

### PR DESCRIPTION
**Pull request by prathameshzarkar9**
Feature name:

Oryx
Description:

This PR adds the following:

Removed old version of dotnet 8.0.1 files which are not required anymore to reduce the size of an image by 6 MB
Changelog:

Updated installation file install.sh
 Checked that applied changes work as expected

### CODE REVIEW

File: src/oryx/install.sh [hunk 1]
```diff
@@ -241,6 +241,7 @@ if [[ "${PINNED_SDK_VERSION}" != "" ]]; then
     MAJOR_MINOR_PATCH1_VERSION=${PINNED_SDK_VERSION%??}
     rm -rf /usr/share/dotnet/shared/Microsoft.NETCore.App/$MAJOR_MINOR_PATCH1_VERSION
```
Comment (samruddhikhandale) [inline src/oryx/install.sh:242]: Can we bump the patch version for this Feature?
https://github.com/devcontainers/features/blob/0d4771c789f0b007b538aaf0922d96f00d574f9c/src/oryx/devcontainer-feature.json#L3

File: src/oryx/install.sh [hunk 2]
```diff
@@ -241,6 +241,7 @@ if [[ "${PINNED_SDK_VERSION}" != "" ]]; then
...
     rm -rf /usr/share/dotnet/shared/Microsoft.AspNetCore.App/$MAJOR_MINOR_PATCH1_VERSION
+    rm -rf /usr/share/dotnet/templates/$MAJOR_MINOR_PATCH1_VERSION
```
Comment (samruddhikhandale) [inline src/oryx/install.sh:244]: Can you add a test check to validate that `/usr/share/dotnet/templates/8.0.1` no longer exists?

**Review by samruddhikhandale — CHANGES_REQUESTED** [review]

File: src/oryx/install.sh (hunk 1 above)
Comment (prathameshzarkar9) [reply src/oryx/install.sh:242]: done

File: src/oryx/install.sh (hunk 2 above)
Comment (prathameshzarkar9) [reply src/oryx/install.sh:244]: done

File: test/oryx/test_python_project.sh [hunk 3]
```diff
@@ -28,5 +28,13 @@ check "oryx-build-python" oryx build --property python_version="${pythonVersion}
 check "oryx-build-python-installed" python3 -m pip list | grep mpmath
 check "oryx-build-python-result" python3 ./src/solve.py
 
+echo "checking 8.0.1 folder is not exist:"
+dirname=/usr/share/dotnet/templates/8.0.1 
+if [ -d "$dirname" ]; then
+    echo "$dirname directory exists"
+else
+    echo "$dirname directory does not exist."
+fi
```
Comment (samruddhikhandale) [inline test/oryx/test_python_project.sh:37]:
```suggestion
check "templates/8.0.1-does-not-exist" ls /usr/share/dotnet/templates/8.0.1 bash -c "test ! -d /usr/share/dotnet/templates/8.0.1"
```

**Review by samruddhikhandale — CHANGES_REQUESTED** [review]

File: test/oryx/test_python_project.sh [hunk 4]
```diff
@@ -28,5 +28,7 @@ check "oryx-build-python" oryx build --property python_version="${pythonVersion}
 check "oryx-build-python-installed" python3 -m pip list | grep mpmath
 check "oryx-build-python-result" python3 ./src/solve.py
 
+check "templates/8.0.1-does-not-exist" ls /usr/share/dotnet/templates/8.0.1 bash -c "test ! -d /usr/share/dotnet/templates/8.0.1"
```
Comment (samruddhikhandale) [inline test/oryx/test_python_project.sh:31]: Hmm.. This isn't working as how I tested in my container. 
Never mind, let's remove this check.

Comment (prathameshzarkar9) [reply test/oryx/test_python_project.sh:31]: Now this is working. I have tested in my container and showing correct check. failing if the folder exists and passing if it is not exists.

**Review by samruddhikhandale — APPROVED** [review]
Nice!

**Pull request merged**